src/routes/tools/local-climate-change-snapshot/index.json.js

```diff
@@ -0,0 +1,31 @@
+import { getToolContent } from "../_common/server-utils";
+import { CLIMATE_CATEGORIES } from "content/tools/local-climate-change-snapshot/categories";
+import { CLIMATE_INDICATORS } from "content/tools/local-climate-change-snapshot/indicators";
+
+export function get(_req, res) {
+  try {
+    const toolIntro = getToolContent(
+      "local-climate-change-snapshot",
+      "tool-intro"
+    );
+
+    res.writeHead(200, { "Content-Type": "application/json" });
+    res.end(
+      JSON.stringify({
+        toolIntro,
+        CLIMATE_CATEGORIES,
```

Comment:
I might alias these here so that they don't need to be passed around as all caps, e.g.

```
climateCategories: CLIMATE_CATEGORIES,
climateIndicators: CLIMATE_INDICATORS,
```

Reply:
good idea. fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/190/commits/499ac6bfea636fe6d8f4e0bb52bc99196050c05c